src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -345,12 +345,13 @@ public List<DataFile> saveAndAddFilesToDataset(DatasetVersion version,
                         StorageIO<DvObject> dataAccess = DataAccess.getStorageIO(dataFile);
                         //Populate metadata
                         dataAccess.open(DataAccessOption.READ_ACCESS);
-                        
+                        // (this will make a remote call to check if the file exists 
+                        // and obtain its size)
```

Comment:
Technically that now happens in the open call above where  there's 
```
            if (isReadAccess) {
                this.setSize(retrieveSizeFromMedia());
```
or similar.